model/ftn/ww3_uprstr.ftn

```diff
@@ -734,8 +770,8 @@
 !/SMC               IX = 1
 !/SMC               IY = ISEA
 !/SMC               IF( SMCWND ) THEN
-!/SMC                  IXW = IX
-!/SMC                  IYW = IY
+!/SMC                  IXW = IY
```

Comment:
+1

Reply:
Agreed - a comment is helpful, I've added the following:

!/SMC                  ! Wind arrays allocated using X,Y convention for regular grids
!/SMC                  ! but override for the SMC grid which is always (NSEA,1)

Reply:
I was more focuseed on the fact that `IXW = IY`, and `IYW = IX`, i.e. they are reversed. It ***looks*** like a bug (i know it isn't) and perhaps it needed a small comment to say "array dimensions swapped - this is right; it's because..."

I know It would be looking at this line suspiciously if I was unfamiliar with this code...

Reply:
Hopefully this is a little clearer??

!/SMC                  ! Wind arrays allocated using (X,Y) convention for regular grids
!/SMC                  ! but overriding here for the SMC grid which are always defined
!/SMC                  ! as (NSEA,1) by switching the IY and IX dimension values around

Reply:
All good! thanks!